internal/usage/usage.go

```
@@ -279,12 +292,105 @@ func (s Store) Reset() error {
+func (s Store) lock() (func(), error) {
+	if err := os.MkdirAll(filepath.Dir(s.Path), 0o755); err != nil {
+		return nil, err
+	}
+	path := s.lockPath()
+	deadline := time.Now().Add(lockWait)
+	var deniedSince time.Time
+	for {
+		err := os.Mkdir(path, 0o755)
+		switch {
+		case err == nil:
+			return func() { os.Remove(path) }, nil
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)